src/modules/robot/Robot.cpp

```diff
@@ -1639,30 +1642,6 @@ bool Robot::append_milestone(const float target[], float rate_mm_s, unsigned int
     // as the last milestone won't be updated we do not actually lose any moves as they will be accounted for in the next move
     if (!auxilliary_move && distance < 0.00001F) return false;
 
-    if (!auxilliary_move) {
```

Comment:
I needed the distance to calculate the mm/sec in G93 mode, and it's calculated further in the code, so I've moved this block down without any changes. shouldn't make any difference, but if it somehow does - let me know.